<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generic Load UI startup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Home">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>181</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Please Click the button below to update the background color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Blue">
     <property name="styleSheet">
      <string notr="true">background-color: blue</string>
     </property>
    </widget>
    <widget class="QWidget" name="Red">
     <property name="styleSheet">
      <string notr="true">background-color: red</string>
     </property>
    </widget>
    <widget class="QWidget" name="Yellow">
     <property name="styleSheet">
      <string notr="true">background-color: yellow</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnYellow">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Yellow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
